src/plugins/janus_streaming.c

```diff
@@ -3394,6 +3421,16 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			json_t *scrypto = json_object_get(root, "srtpcrypto");
 			json_t *e2ee = json_object_get(root, "e2ee");
 			json_t *pd = json_object_get(root, "playoutdelay_ext");
+			json_t *abscaptime_src_id = json_object_get(root, "abscapturetime_src_ext_id");
+			if(abscaptime_src_id && (json_integer_value(abscaptime_src_id) < 1 || json_integer_value(abscaptime_src_id) > 14)) {
+				JANUS_LOG(LOG_ERR, "Can't add 'rtp' stream, invalid abscaptime_src_id value...\n");
+				error_code = JANUS_STREAMING_ERROR_CANT_CREATE;
```

Comment:
The error here should be `JANUS_STREAMING_ERROR_INVALID_ELEMENT`, and the description something like `Invalid element (abscaptime_src_id must be an integer between 1 and 14)`.

Reply:
Fixed